firefox_ui_harness/runtests.py

```diff
@@ -102,5 +60,9 @@ def cli(runner_class=FirefoxUITestRunner, parser_class=FirefoxUIOptions):
         sys.exit(10)
 
 
+def cli_update():
```

Comment:
You cannot do this. We need specific methods for every entry point, which gets added. You cannot specify the runner class and parser class on the setup.py level.